cmd/protoc-gen-openapi/generator/openapi-v3.go

```diff
@@ -951,9 +951,9 @@ func (g *OpenAPIv3Generator) addSchemasToDocumentV3(d *v3.Document, messages []*
 					Value: &v3.SchemaOrReference{
 						Oneof: &v3.SchemaOrReference_Schema{
 							Schema: &v3.Schema{
-								Description: messageDescription,
+								Description: "Represents a dynamically typed value which can be either null, a number, a string, a boolean, a recursive struct value, or a list of values.",
+								Nullable:    true, // this only works with OpenAPI 3.0.x
```

Comment:
Is it possible to set this to `required` as well?
Then it would adhere to both OpenAPI v3.0.x and the protobuf description, right?